### DISCUSSION TITLE
Ability to sort organizers list

### DISCUSSION THREAD

**Discussion by djredman99** [Reetus/ClassicAssist#281] [Ideas]
**Is your feature request related to a problem? Please describe.**
The organizers you create are permanently listed in the order you create them.

**Describe the solution you'd like**
Having the ability to drag/drop the organizers in the desired order would be ideal.

**Describe alternatives you've considered**
An option to sort alphabetically or by original order (be able to go back and forth)

**Additional context**
none

**Comment by djredman99** [comment]
I have discovered that I have the ability to edit the profile file to get it in the order that I like, but this is less than ideal.  Certainly not a high priority issue, but a nice-to-have.

**Comment by Reetus** [comment]
Implemented in https://github.com/Reetus/ClassicAssist/releases/tag/0.4.410
